AirCasting/SDSync/ViewsFlow/BackendSyncCompleted/BackendSyncCompletedView.swift

```diff
@@ -52,16 +52,28 @@ private extension BackendSyncCompletedView {
     
     var continueButton: some View {
         Button {
-            viewModel.presentNextScreen = true
+            switch viewModel.sessionNextStep() {
+            case .bluetooth: viewModel.procedToBTScreen()
+            case .restart: viewModel.procedToRestartScreen()
+            }
```

Comment:
this logic should be inside viewmodel. Here it should look more like:
```
Button { viewModel.continueButtonTapped() }
```

and inside ViewModel it decides what to do next:
```
func continueButtonTapped() {
    switch self.nextStep() {
    ...
    }
}